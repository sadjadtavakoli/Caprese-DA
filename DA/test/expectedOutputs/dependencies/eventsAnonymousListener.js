{"arrowFunction-test/unit_tests/eventsanonymouslistener.js-5-7":{"callers":["t_1"],"tests":[]},"keyMap":{"t_1":"arrowFunction-test/unit_tests/eventsanonymouslistener.js-1-10","t_a37":"arrowFunction-test/unit_tests/eventsanonymouslistener.js-5-7"}}