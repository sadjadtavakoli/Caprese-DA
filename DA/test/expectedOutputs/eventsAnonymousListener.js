
#1 function require is called with variables args by eventsAnonymousListener.js
#2 class EventEmitter's constructor is called with variables args by eventsAnonymousListener.js
#9 function eventsAnonymousListener.js emitted event connection of EventEmitter
#5 function anonymous5 entered with variables args throught event connection emitted by function eventsAnonymousListener.js
#5 function anonymous5 exited
#10 function eventsAnonymousListener.js emitted event connection2 of EventEmitter
#end Execution