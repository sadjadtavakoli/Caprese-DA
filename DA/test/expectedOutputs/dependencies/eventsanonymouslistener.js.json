{"eventsanonymouslistener.js-test/unit_tests/eventsanonymouslistener.js-1-10":{"callers":[],"tests":["t_1"]},"arrowFunction-test/unit_tests/eventsanonymouslistener.js-5-7":{"callers":[],"tests":["t_1"]},"keyMap":{"t_1":"eventsanonymouslistener.js-test/unit_tests/eventsanonymouslistener.js-1-10","t_4":"arrowFunction-test/unit_tests/eventsanonymouslistener.js-5-7"}}